sphinx/util/docutils.py

```diff
@@ -32,8 +32,14 @@
     from sphinx.config import Config
     from sphinx.environment import BuildEnvironment
 
-
-__version_info__ = version.parse(docutils.__version__).release
+deprecated_alias('sphinx.util.docutils',
+                 {
+                     '__version_info__': docutils.__version_info__,
+                 },
+                 RemovedInSphinx60Warning,
```

Comment:
We're not in hurry. So I'll remove this in 7.0. I'll change this after merging.

Reply:
Oops, I meant to change that, sorry.

A